grapher/controls/CollapsibleList.scss

```diff
@@ -0,0 +1,15 @@
+.collapsibleList {
```

Comment:
I think we had a team conversation about CSS class naming a while back, and went with camelCase, but I'm wondering if we should revisit that convo as a team and maybe switch to `CSSClassName === TypeScriptClassName`

Reply:
I agree, I think it makes to just use the TypeScriptClassName. Will change when we decide that.